tripy/trace-inv.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Will need to remove this file before merging.

Reply:
yes

Reply:
removed